src/Pages/Login.jsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import GoogleLogin from "../library/components/GoogleLogin";
+import Logo from "../library/components/Logo";
+import CLabel from "../library/components/Label";
+import Content from "../library/components/Content";
+
+const Login = () => {
+  return (
+    <Content className="MainContainer" >
+        <Content style={{ margin: "3rem 0 2rem 0", textAlign: "center" }}>
+      <Logo width={243} height={50}/>
+      <CLabel>
+        <h6 style={{ fontStyle: "italic", margin: 0 }}>
+          <em>We are at your service</em>
+        </h6>
+      </CLabel>
+      </Content>
+      <Content className="Container"
```

Comment:
You are using css class Container and inline style. Lets stick to one.

Reply:
Done